interfaces/builtin/basedeclaration.go

```diff
@@ -161,6 +161,11 @@ slots:
       slot-snap-type:
         - core
     deny-auto-connection: true
+  autopilot-introspection:
+    allow-installation:
+      slot-snap-type:
+        - core
+    deny-auto-connection: true
```

Comment:
This base declaration is good. This means that a snap may 'plugs: [ autopilot-introspection ]' but only have the security policy added when the user does a manual connection.